data/pc/latest/proto.yml

```diff
@@ -3689,8 +3096,12 @@
       actionId: varint
       jumpBoost: varint
    # MC: ServerboundPlayerInputPacket
-   packet_player_inputs:
-      inputsBitset: u8
+   packet_steer_vehicle:
```

Comment:
This name doesn't make sense anymore since this packet is sent even if the player isn't in a vehicle